Sources/Client/Model/Channel/Channel.swift

```diff
@@ -196,9 +199,40 @@ public final class Channel: Codable {
         unreadMessageReadAtomic.set(messageRead)
         unreadCountAtomic.set(.noUnread)
     }
+}
+
+extension Channel: Hashable, CustomStringConvertible {
+    
+    public static func == (lhs: Channel, rhs: Channel) -> Bool {
+        lhs.cid == rhs.cid
+    }
+    
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(cid)
+    }
     
-    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases),
+    public var description: String {
+        let opaque: UnsafeMutableRawPointer = Unmanaged.passUnretained(self).toOpaque()
```

Comment:
You can just use `ObjectDescription` since `Channel` is a class

Reply:
Please show me an example.

Reply:
```swift
public var description: String {
  let opaque = ObjectIdentifier(self)
  return "Channel<\(opaque)>:\(cid):\(name ?? "<NoName>")"
```
On a second check, this prints `ObjectIdentifier(address)` instead of just `address`, so the current way is better 👍🏼